packages/ceramic-account/src/accountLinks.ts

```diff
@@ -0,0 +1,27 @@
+import type { CeramicApi, CeramicDocument } from './types'
+
+class AccountLinks {
+  private _accountLinkDocuments: Record<string, CeramicDocument>;
+
+  constructor (public ceramicDoc: CeramicDocument, private _ceramic: CeramicApi, public provider?: any) {
+    // Keep all account link documents open to receive updates
+    this._accountLinkDocuments = {}
+  }
+
+  // The following can be implemented similarly to
+  // https://github.com/3box/3box-js/blob/ceramic/src/accountLinks.js
+
+  async getAllAddresses () { }
+
+  async linkAddress (address: string, proof?: string) { }
```

Comment:
@oed should this be something like `setAddressLink` to be more clear? And below it could also be `removeAddressLink`?

Reply:
Also I noticed we're inconsistent between AddressLink and AccountLink. We should standardize on AccountLink, no?